### DISCUSSION TITLE
Are the parent commits of a commit accessible?

### DISCUSSION THREAD

**Discussion by kayman-mk** [python-gitlab/python-gitlab#2705] [Q&A] [answered]
## Description of the problem, including code/CLI snippet

I was looking for the ids of the parent commits for a specific commit. I know that GitLab returns them as `parent_ids` when calling `GET /projects/:id/repository/commits/:sha`. But I wasn't able to find them in the `ProjectCommit` class.

Did I overlook them or do we have to add them to `ProjectCommit` as they are missing at the moment?

## Specifications

  - python-gitlab version: v4.0.0
  - API version you are using (v3/v4): v4
  - Gitlab server version (or gitlab.com): not relevant

**Comment by nejch** [comment, marked as answer]
@kayman-mk yes, if the API returns it then this should dynamically be added as the `commit.parent_ids` attribute. We do not actually track all the possible attributes GitLab returns, as it would be a lot to maintain.

Please reopen this if you have any issues with the endpoint, but generally just give it a try and see - you can also print out `.attributes` to see the entire response dict.

**Comment by kayman-mk** [comment]
Works like a charm. Really good project. Thanks for your work, guys!
